seDataDevices.py

```diff
@@ -214,7 +214,7 @@ def parseDevTable(self, data):
                     )
 
         if self.defnLen > devLen :
-            raise ValueError('You have defined more bytes, {}, than the message contains, {}'.format(self.defnLen, devLen))
+            log('You have defined more bytes, {}, than the message contains, {}\nYou might get some type of runtime error!\nOr just rubbish output'.format(self.defnLen, devLen))
```

Comment:
I agree.  It should only happen during development of a new `ParseDevice` subclass (or if solaredge ever redefine and shorten an existing message :-( ).

I have just pushed a commit to revert to raising a ValueError.